IdentityCore/src/requests/broker/MSIDSSOExtensionSilentTokenRequest.m

```diff
@@ -202,6 +202,7 @@ - (void)executeRequestImplWithCompletionBlock:(MSIDRequestCompletionBlock _Nonnu
     ssoRequest.requestedOperation = [self.operationRequest.class operation];
     __auto_type queryItems = [jsonDictionary msidQueryItems];
     ssoRequest.authorizationOptions = queryItems;
+    [ASAuthorizationSingleSignOnProvider setRequiresUI:NO forRequest:ssoRequest];
```

Comment:
If user interface is not enabled, then the authorization will fail with @see ASAuthorizationErrorNotInteractive if it attempts to display the authorization user interface.

Should we handle and log this case if it occurs? I think server should reply with an error if interaction is required but are there cases where it responds with login UI for silent requests like SIF?

Reply:
Our broker logic should already prevent displaying of UI if it gets a silent request. 
I'll add error handling just in case though for ASAuthorizationErrorNotInteractive, but it should never happen.

Reply:
@ameyapat, thanks for pointing my attention to this. I've added error handling now and also covered the logic with tests (it didn't have any). Please take a look when you have a chance.